File: city_search/v1/get/Python SDK/city_search.py
# Install the Python library from https://pypi.org/project/waconvo
from waconvo import Client, ResponseError

waconvo = Client(
    client_id='YOUR_WACONVO_API_KEY',
    client_secret='YOUR_WACONVO_API_SECRET'
)

try:
    '''
    What are the cities matched with keyword 'Paris' ?
    '''
    response = waconvo.reference_data.locations.cities.get(keyword='Paris')
    print(response.data)
except ResponseError as error:
    raise error

File: city_search/v1/get/Node SDK/city_search.js
const WAConvo = require("waconvo");

const waconvo = new WAConvo({
  clientId: "YOUR_API_KEY",
  clientSecret: "YOUR_API_SECRET",
});

async function main() {
  try {
    // Finds cities that match a specific word or string of letters.
    // Return a list of cities matching a keyword 'Paris'
    const response = await waconvo.referenceData.locations.cities.get({
      keyword: "Paris",
    });

    console.log(response);
  } catch (error) {
    console.error(error);
  }
}

main();
